README.md

```diff
@@ -174,6 +174,18 @@ implicit val residentFormat = Json.format[Resident]
 
 With the `Reads` and/or `Writes` in scope, I can then easily convert my class using `toJson` and `fromJson`
 
+In case you don't want to use an Option but a sensible default value you can use case class default parameters
```

Comment:
```
an `Option`
```

Reply:
", you can define default values for case class."

Reply:
That doesn't sound right